test/reuse/util/userSettings/setDateFormatFromUserSettings.spec.js

```diff
@@ -0,0 +1,38 @@
+const data = require("./data/data.json");
+
+describe("userSettings", function () {
+  const DATE_FORMAT_LENGTH = 10;
+
+  describe("setDateFormatFromUserSettings.spec - passing date format", function () {
+
+    it("Preparation: Set systemUrl ", async function () {
+      browser.config.params.systemUrl = util.data.decrypt(data.systemUrl);
+    });
+
+    it("Execution & Verification: Set User Date", async function () {
+      await util.userSettings.setDateFormatFromUserSettings(util.data.decrypt(data.username), util.data.decrypt(data.password));
+      const date = await common.date.getToday("yyyy/mm/dd");
+      const userDate = await common.date.getToday(process.env.USER_SETTINGS_DATE_FORMAT);
+      //Ensure returned date matches 10 digits
+      common.assertion.expectEqual(date.length, DATE_FORMAT_LENGTH);
```

Comment:
expect type of enum dateformat object instead of length

Reply:
same as above as discussed yesterday

Reply:
as discussed js can not handle types as we expected in our meeting, therefore keeping the date evaluation only.